src/jit/emitxarch.cpp

```diff
@@ -171,6 +171,7 @@ bool emitter::IsDstDstSrcAVXInstruction(instruction ins)
         case INS_vperm2i128:
         case INS_xorpd:
         case INS_xorps:
+        case INS_pmuldq:
```

Comment:
nit: It would be nice to keep this list in alphabetical order

Reply:
Thanks, will fix.